cmd/notary-signer/main.go

```diff
@@ -191,7 +204,7 @@ func main() {
 		log.Println("HTTP server listening on", httpAddr)
 	}
 
-	err = server.ListenAndServeTLS(certFile, keyFile)
+	err = server.ListenAndServe()
```

Comment:
Ah ok, thanks!  I should have just read the golang function for both of these.  Is it preferable to call `ListenAndServeTLS` with the two blank lines, or just to set up the listener and call `Serve`, in this case?